corehq/apps/userreports/templates/userreports/edit_data_source.html

```diff
@@ -19,6 +19,10 @@ <h4>
       </h4>
       {% trans "If rebuilt again next rebuild will only start when the previous rebuild(s) finishes." %}
     </div>
+  {% elif data_source.meta.build.awaiting %}
+    <div class="alert alert-info">
+      {% trans "Please note that this datasource is yet be picked for build / rebuild by CommCare HQ." %}
```

Comment:
Very nit: I think you can leave the "picked for" language so it will just read:
"Please note that this datasource is yet be build / rebuild by CommCare HQ"

Reply:
Nice, that makes sense. I will update.

Reply:
https://github.com/dimagi/commcare-hq/pull/35633/commits/8c62cd2d4ee54d5dea3a9d3ded1fedc92b637258